dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1218,6 +1218,17 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_HDFS_TRASH_ENABLE =
+      booleanBuilder(Name.UNDERFS_HDFS_TRASH_ENABLE)
+          .setDefaultValue(false)
+          .setDescription("Whether to enable the HDFS trash feature, "
+              + "it is important to note that after enabling this configuration, "
+              + "you need to set the relevant trash configurations (such as 'fs.trash.interval') "
+              + "in 'hdfs-site.xml' and 'core-ste.xml' "
+              + "in order to truly use the HDFS trash functionality.")
```

Comment:
Suggested rewording:

If enabled, when a user deletes a file that's stored in HDFS via Alluxio, Alluxio will make use of HDFS's trash and try to move it to the trash directory instead of deleting it directly from HDFS. Note that HDFS must be configured to enable the trash functionality. If trashing is not enabled in HDFS, Alluxio will delete the file directly, regardless of this configuration.

Reply:
OK